lib/embeds/widgets/youtube-video-app.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
 import 'package:url_launcher/url_launcher.dart';
-import 'package:youtube_player_flutter/youtube_player_flutter.dart';
+import 'package:youtube_player_flutter_quill/youtube_player_flutter_quill.dart';
```

Comment:
I'm struggling to see where's the source code for this repository. I don't trust the library. Can you explain more what's the story?